addon/templates/components/frost-modal-input.hbs

```diff
@@ -1,11 +1,45 @@
-{{#if titleComponent}}
-  {{component titleComponent}}
-{{else}}
-  <div class="input-header">
-    <div class="primary-title">{{title}}</div>
-    {{#if subtitle}}
-      <div class="sub-title">{{subtitle}}</div>
-    {{/if}}
-  </div>
-{{/if}}
-{{yield}}
+{{#ember-remodal
+  buttonClasses='frost-button'
+  forService=true
+  closeOnOutsideClick=false
+  onOpen=(action 'modalOpen')
+  onClose=(action 'modalClose')
+  closeOnEscape=false
+  disableNativeClose=true
+  name=modalName as |modal|}}
+  {{yield}}
+  {{#modal.open}}
+    {{#yield-slot 'openButton'}}
+      {{yield (block-params (hash
+        button=(component 'frost-button')))}}
+    {{/yield-slot}}
+  {{/modal.open}}
+
+  {{#if titleComponent}}
+    {{component titleComponent}}
```

Comment:
Wasnt sure if we wanted changes in the header...just basing it off usage in MCP. The only difference I see is in the padding. We have left and right padding here for the header element but info-bar only has left padding. Guessing its due to the optional action button.